src/v/kafka/group_probe.h

```diff
@@ -82,10 +83,31 @@ class group_offset_probe {
              sm::description("Consumer group committed offset"),
              labels)
              .aggregate({sm::shard_label})});
+
+        _public_metrics.add_group(
+          prometheus_sanitize::metrics_name("kafka:consumer:group"),
+          {sm::make_gauge(
+             "committed_offset_timestamp_seconds",
+             [this] { return _commit_timestamp.value()/1000; },
```

Comment:
I think this will fail the lint check. There's a `.clang-format` file in the root of the repository.